src/ralph/backends/data/async_mongo.py

```diff
@@ -280,8 +268,19 @@ async def close(self) -> None:
             self.logger.error(msg, error)
             raise BackendException(msg % error) from error
 
+    def _get_target_collection(
+        self, target: Union[str, None]
+    ) -> AsyncIOMotorCollection:
+        """Return the target collection."""
+        try:
+            return self.database[target] if target else self.collection
+        except InvalidName as error:
+            msg = "The target=`%s` is not a valid collection name: %s"
+            self.logger.error(msg, target, error)
+            raise BackendParameterException(msg % (target, error)) from error
```

Comment:
It was already covered by the tests, right?

Reply:
Well spotted) Thanks) There was indeed a missing test for the write method with an invalid `target`) added it)